Orange/widgets/utils/tests/test_state_summary.py

```diff
@@ -183,6 +184,15 @@ def test_details(self):
                   f'Target: —'
         self.assertEqual(details, format_summary_details(data))
 
+        data = Table.from_numpy(domain=None, X=np.random.random((10000, 1000)))
+        details = f'{len(data):n} instances, ' \
+                  f'{len(data.domain.variables)} variables\n' \
+                  f'Features: {len(data.domain.variables)} numeric \n' \
+                  f'Target: —'
+        with patch.object(Table, "get_nan_frequency_attribute") as mock:
+            self.assertEqual(details, format_summary_details(data))
+            self.assertFalse(mock.called)
```

Comment:
The usual way to test this is `mock.assert_not_called()` because it gives a better diagnostic message - if you test for `False`, it just tells you that `True` is not `False`...